src/EventService.Business/Commands/Category/FindCategoryCommand.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using LT.DigitalOffice.EventService.Business.Commands.Category.Interfaces;
+using LT.DigitalOffice.EventService.Models.Dto.Requests.Category;
+using LT.DigitalOffice.EventService.Models.Dto.Responses.Category;
+using LT.DigitalOffice.EventService.Validation.Category.Interfaces;
+using MassTransit;
+using Microsoft.AspNetCore.Http;
+using ValidationResult = FluentValidation.Results.ValidationResult;
+
+namespace LT.DigitalOffice.EventService.Business.Commands.Category;
+
+public class FindCategoryCommand : IFindCategoryCommand
+{
+    private IRequestClient<FindCategoryRequest> _request;
+    private IFindCategoryValidator _validator;
+    private readonly IHttpContextAccessor _contextAccessor;
+
+    public FindCategoryCommand(
+      IFindCategoryValidator validator, 
+      IRequestClient<FindCategoryRequest> request,
+      IHttpContextAccessor contextAccessor)
+    {
+        _validator = validator;
+        _request = request;
+        _contextAccessor = contextAccessor;
+    }
+    
+    public async Task<FindCategoryResponse> ExecuteAsync(FindCategoryRequest request)
```

Comment:
Will be better use here for return value Task<OperationResultResponse<List<CategoryInfo>>
Thus you may clean find endpoint from check on success